src/ContactPage/Contact.scss

```diff
@@ -2,45 +2,51 @@
 @import "../styles/layout";
 
 .contact-section {
+  background-image: url(../../assets/letters.jpg);
+  
+  
+  height:calc(100vh - 60px);
+  background-size: cover;
   display: flex;
   flex-direction: column;
   flex-wrap: wrap;
 
   &__border {
     padding: 2% 0;
-    background: rgba(42, 71, 110, 0.918);
     width: 100%;
+    background-color: grey;
+    opacity: 0.9;
+    margin-top: 8%;
   }
   &__title {
     font-family: $base-font-family;
-    margin: 0 5%;
-    margin-bottom: 10px;
-    color: rgb(212, 212, 212);
+    margin: 2% 5% 3% 5%;
```

Comment:
You've given each of the elements in your container their own horizontal margin, would be easier just to put horizontal padding on the container so all elements inside are aligned